source/NVDAObjects/IAccessible/sysListView32.py

```diff
@@ -228,13 +229,31 @@ def _get_columnCount(self):
 			return 1
 		return count
 
-	def _get__columnOrderArray(self):
-		coa=(c_int *self.columnCount)()
+	def _getColumnOrderArrayRaw(self, columnCount: int):
+		coa = (ctypes.c_int * columnCount)()

```

Comment:
Could this variable be renamed to be clearer?

```suggestion
		columnOrderArray = (ctypes.c_int * columnCount)()
```